### ISSUE TITLE
Obtain initial point clouds on the custom dataset

### ISSUE THREAD

**Issue by Daiqy** [yizhangphd/FreqPCR#5]
Hi! Thank you for your excellent work. 

You mention that the initial Nerf-synthetic points should be genertated from [Point-NeRF](https://github.com/Xharlie/pointnerf). So for the custom dataset, I'm wondering how to obtain those initial point clouds. Should we train Point-NeRF at first, and then make a inference to generate them? 

Thank you!

**Comment by seanywang0408** [comment]
No, you don't need to train Point-NeRF to obtainn point clouds. Nor did we. Just use their trained MVSNet to obtain point clouds, as instructed in https://github.com/Xharlie/pointnerf#point-initialization--generalization.

**Comment by Daiqy** [comment]
Thanks for your quick reply! 

I have checked the point initialization instruction of PointNeRF, and it seems they only provide the script of inference on DTU dataset. Since our custom dataset is similar to NeRF Synthetic dataset, and the parameters are quite different between DTU and NeRF Synthetic, could you kindly provide the scripts or some guidance for generating initial point clouds on NeRF Synthetic dataset?

Thanks again!
